src/graphql/schema.gql

```diff
@@ -210,6 +210,9 @@ input VoteWhere {
   vp_lte: Float
   vp_state: String
   vp_state_in: [String]
+  choice: Any
+  choice_in: [Any]
+  choice_match: [String]
```

Comment:
`choice_match` can still search exact match right? if so we don't need `choice` and `choice_in`

Reply:
if not, we should match the exact match